controller.js

```diff
@@ -0,0 +1,29 @@
+const express = require("express");
+const app = express(); //initializes the app
+const PORT = 8080;
+
+const databaseService = require("./service.js");
+const storager = new databaseService(); //instantiates a DatabaseService object
```

Comment:
![image](https://user-images.githubusercontent.com/10796765/169837064-d9f0cf9a-af66-4ffb-a18a-e4316f3bb3aa.png)
This is the old request. only `storager` is a variable here. Class names (`DatabaseService`) should remain PascalCased